<launch>
	<node pkg="sunray_swarm" name="swarm_with_obstacles" type="swarm_with_obstacles" output="screen">
		<rosparam command="load" file="$(find sunray_swarm)/launch_sim/ugv_params.yaml" />
	</node>
</launch>